### ISSUE TITLE
404

### ISSUE THREAD

**Issue by BabyRdx** [ScriptSmith/reaper#12]
I setup 3 keys following your guide; Twitter, FB and Pinterest. 

When I try to scrape Pinterest or FB I get the following popup error.

>   "message": "404 Not Found: The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.", 
>   "type": "http"
I tried resetting the Pinterest API key as per your instructions, same thing.

Twitter key works fine.

**Comment by ScriptSmith** [comment]
This is likely because you are requesting information from pages / users that you don't have access to.  When I first published Reaper, social media APIs gave much more access to public data than they do now.

**Pinterest**
You should be able to access your own information using `me` as your username. You'll need an app that's been reviewed and approved by pinterest to access other data.

**Facebook**
I don't think you're getting a 404 error for Facebook, you're probably getting a 400 error which contains a message explaining what went wrong. Make sure you scroll up and down in reaper's error manager, and you might want to clear the log between requests by clicking `Options` -> `Clear` in the error manager window.

**Comment by BabyRdx** [comment]
Regarding Pinterest I will try "app reviewed" but I don't think I will get far.

Retried Facebook again.  Trying to scrape a single user which when logged into my account I can access.  I replaced the access token, user name to scrape and the fbtraceID with placeholders, 

Error I got is:
`404 Client Error: Not Found for url: https://graph.facebook.com/v2.9/TARGET_USERNAME_FROM_URL/?access_token=ACCESS_TOKEN
{"error":{"message":"(#803) Cannot query users by their username (TARGET_USERNAME_FROM_URL)","type":"OAuthException","code":803,"fbtrace_id":"SOME_STRING_WAS_HERE"}}
Retrying in 5 seconds`
